TKeazirian.HTTPServer.Tests/ParserTests.cs

```diff
@@ -6,14 +6,55 @@ public class ParserTests
 {
     private const string NewLine = "\r\n";
 
-    [Fact]
-    public void CanParseBody()
+    [Theory]
+    [InlineData($"Hello{NewLine}how{NewLine}are{NewLine}you")]
+    [InlineData("")]
+    public void CanParseBody(object value)
     {
-        string expectedBody = $"Hello{NewLine}how{NewLine}are{NewLine}you";
+        object expectedBody = value;
         string testRequest = $"HTTP/1.1 200 OK{NewLine}{NewLine}{expectedBody}";
```

Comment:
this looks like a Response and not a body

Reply:
Worked on a bit of a refactor with Travis so I'll be pushing up some code that will address this!